modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/AgentUtils.java

```diff
@@ -323,6 +328,20 @@ public static void send(Session ses, WebSocketResponse evt, long timeout, TimeUn
         }
     }
 
+    /**
+     * @param s String to split.
+     * @return List of not empty items.
+     */
+    public static List<String> split(String s) {
+        if (F.isEmpty(s))
+            return Collections.emptyList();
+
+        return Arrays.stream(s.trim().split(","))
+                .map(String::trim)
+                .filter(item -> !F.isEmpty(item))
+                .collect(toList());
```

Comment:
I recommend to use `toSet()` collector to avoid duplicates which are very much possible with tokens